swap_meet/vendor.py

```diff
@@ -1,2 +1,87 @@
+#Wave 1
+# from item import Item
 class Vendor:
-    pass
+    def __init__(self, inventory = []):
+        self.inventory = inventory
+    def add(self,item):
+        self.inventory.append(item)
+        return item 
+    def remove(self,item):
+        if item not in self.inventory:
+            return False
+
+        self.inventory.remove(item)
+        return item 
```

Comment:
should be an if/else statement 
```suggestion
       else:
              self.inventory.remove(item)
              return item 
```